scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -21,20 +21,37 @@ function prepare_vm {
     kernel_version="$3"
     rhel_username="$4"
     rhel_password="$5"
-    
+    ret=0
+
     subscription-manager register --force --username "${rhel_username}" --password "${rhel_password}"
     subscription-manager attach --auto
     subscription-manager release --set=${os_version}
     subscription-manager repos --enable=${kernel_repo}
     yum clean all
-    yum -y install wget gcc
+    yum -y install wget gcc redhat-lsb-core
     yum -y install kernel-${kernel_version}
```

Comment:
you can do the if directly on the yum install without additional variable.
https://github.com/koalaman/shellcheck/wiki/SC2181

Reply:
Fixed